[2023-03-29 19:35:27,688] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: data_merge_dag.merge_task scheduled__2023-03-29T10:05:00+00:00 [queued]>
[2023-03-29 19:35:27,697] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: data_merge_dag.merge_task scheduled__2023-03-29T10:05:00+00:00 [queued]>
[2023-03-29 19:35:27,697] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2023-03-29 19:35:27,697] {taskinstance.py:1357} INFO - Starting attempt 3 of 4
[2023-03-29 19:35:27,698] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2023-03-29 19:35:27,711] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): merge_task> on 2023-03-29 10:05:00+00:00
[2023-03-29 19:35:27,714] {standard_task_runner.py:52} INFO - Started process 227 to run task
[2023-03-29 19:35:27,717] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'data_merge_dag', 'merge_task', 'scheduled__2023-03-29T10:05:00+00:00', '--job-id', '672', '--raw', '--subdir', 'DAGS_FOLDER/data_merge.py', '--cfg-path', '/tmp/tmpushe4fpm', '--error-file', '/tmp/tmpuwbsi_ab']
[2023-03-29 19:35:27,717] {standard_task_runner.py:80} INFO - Job 672: Subtask merge_task
[2023-03-29 19:35:27,763] {task_command.py:369} INFO - Running <TaskInstance: data_merge_dag.merge_task scheduled__2023-03-29T10:05:00+00:00 [running]> on host 491e5e6b4ad4
[2023-03-29 19:35:27,836] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=kartaca
AIRFLOW_CTX_DAG_ID=data_merge_dag
AIRFLOW_CTX_TASK_ID=merge_task
AIRFLOW_CTX_EXECUTION_DATE=2023-03-29T10:05:00+00:00
AIRFLOW_CTX_TRY_NUMBER=3
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-03-29T10:05:00+00:00
[2023-03-29 19:35:27,885] {python.py:173} INFO - Done. Returned value was: None
[2023-03-29 19:35:27,897] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=data_merge_dag, task_id=merge_task, execution_date=20230329T100500, start_date=20230329T193527, end_date=20230329T193527
[2023-03-29 19:35:27,930] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-03-29 19:35:27,964] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
